hoshino/modules/weibo/weibo.py

```diff
@@ -1,51 +1,42 @@
 # -*- coding: UTF-8 -*-
 
 import json
-import os
 import random
 import sys
-import traceback
 from collections import OrderedDict
 from datetime import date, datetime, timedelta
 from time import sleep
 
-import httpx as requests
+import httpx
 from lxml import etree
-from hoshino.service import Service, Privilege as Priv
-from hoshino import util, logger
-from hoshino.res import R
+from hoshino import logger
+from .exception import *
 
-sv = Service('weibo-poller', use_priv=Priv.ADMIN, manage_priv=Priv.SUPERUSER, visible=False)
-user_configs = util.load_config(__file__)
-'''
-sample config.json
-
-[{
-    "user_id": "6603867494",
-    "service_name": "pcr-weibo",
-    "filter": true
-}]
-'''
-
-class Weibo(object):
+class WeiboSpider(object):
     def __init__(self, config):
         """Weibo类初始化"""
         self.validate_config(config)
-        self.filter = config['filter']  
-        self.user = self.get_user_info(config["user_id"])
-
+        self.filter = config['filter'] 
+        self.user_id = config['user_id']
+        self.user = self.get_user_info(self.user_id)
```

Comment:
The only sync requests is in the constructor. 
I would be glad If there's any better way to do so.